buildSrc/src/main/java/org/elasticsearch/gradle/vagrant/VagrantMachine.java

```diff
@@ -87,14 +91,20 @@ public void execute(Action<VagrantExecSpec> action) {
                 execSpec.args(Arrays.asList(vagrantSpec.args));
             }
 
+            // output from vagrant needs to be manually curated because --machine-readable isn't actually "readable"
             UnaryOperator<String> progressHandler = vagrantSpec.progressHandler;
             if (progressHandler == null) {
                 progressHandler = new VagrantProgressLogger("==> " + extension.getBox() + ": ");
             }
-            OutputStream output = execSpec.getStandardOutput();
-            // output from vagrant needs to be manually curated because --machine-readable isn't actually "readable"
-            OutputStream progressStream = new ProgressOutputStream(vagrantSpec.command, progressHandler);
-            execSpec.setStandardOutput(new TeeOutputStream(output, progressStream));
+
+            execSpec.setStandardOutput(new ProgressOutputStream(vagrantSpec.command, progressHandler, System.out));
+            try {
+                execSpec.setErrorOutput(
+                    Files.newOutputStream(project.getBuildDir().toPath().resolve("vagrant.log"), CREATE, APPEND)
```

Comment:
LoggedExec has an option to spool the output to disk (although we may need to convert the static exec method to support that option).